src/probly/transformation/ensemble/flax.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[119]:
+
+
+from flax import nnx
+import jax
+import jax.numpy as jnp
+import numpy as np
+import warnings 
+warnings.filterwarnings("ignore")
+
+
+# In[141]:
+
```

Comment:
1. Please do not ignore warnings without a good reason.
2. Always provide a docstring at the top of each module.
3. Do not commit shebangs or `# In[xxx]` comments used during development.

```suggestion
"""TODO"""

from __future__ import annotations

from flax import nnx
import jax
import jax.numpy as jnp
import numpy as np
```